docs/overview/changelog.md

```diff
@@ -15,19 +15,44 @@ log][commit-log] for an up-to-date list of raw changes.
 
 - New [resolve topic](https://zulip.com/help/resolve-a-topic) feature
   allows marking topics as completed for lightweight workflow management.
+- New [web-public streams](https://zulip.com/help/web-public-streams)
+  allows Zulip organizations to enable visitors to browse certain
+  streams without creating an account.
 - Users can now select an emoji when setting an away status.
 - OpenID Connect joins SAML, LDAP, Google, GitHub, Azure Active
   Directory, and more as a supported Single Sign-On provider.
+- Added SCIM integration for synchonizing accounts with an external
+  user database.
 - Redesigned "Stream settings" to be much more usable, with separate
-  tabs for personal settings, global settings, and membership.
+  tabs for personal settings, global settings, and membership, and
+  more consistent style with the rest of Zulip's settings.
+- Stream creation was redesigned with a much cleaner interface,
+  especially for selecting initial subscribers.
 - Redesigned "Full user profile" widget to show the user's stream
   and user group subscriptions.
 - Reorganized personal and organization settings to have clearer
   labels and make it easier to find privacy settings.
+- Organization administrators can now configure the default personal
+  preference settings for new users joining the organization.
 - Migrated most permissions settings that only supported
-  administrators or normal users to fully support roles.
+  administrators or normal users to support choosing which roles have
+  permission.
```

Comment:
Umm, not sure about this but -- 

Can we go with something like -- 
"Migrated most `of` the permissions settings that only ... "

instead of 

"Migrated most permissions settings that only ...."